docs/Manual/04 Fusing.md

```diff
@@ -202,16 +203,16 @@ def _():
 
 schedule1 = nest1.create_schedule()
 ```
-In both `schedule0` and `schedule1`, the first dimension corresponds to the rows of `C` and the second dimension corresponds to the columns of `C`. In addition, `schedule0` has a third dimension that `schedule1` does not have. Therefore, we fuse the first two dimensions of the iteration spaces and leave the third dimension of `schedule0` unfused.
+In `schedule0` and `schedule1`, the first dimension represents the rows of `C` and the second dimension represents the columns of `C`. Additionally, `schedule0` has a third dimension that `schedule1` does not have. Therefore, we fuse the first two dimensions of the iteration spaces and leave the third dimension of `schedule0` unfused.
 ```python
 schedule = acc.fuse((schedule0, schedule1), partial=2)
 f, i, j, k0 = schedule.get_indices()
 ```
-The fused iteration space `schedule` has a shape of (2, 16, 10, 11), its slice (0, \*, \*, \*) contains a copy of `schedule0`, its slice (1, \*, \*, 0) contains a copy of `schedule1`, and the rest of its elements are filled with padding. Note that the code above overwrites the index `k0`: originally, `k0` was an index of `schedule0`, but now it is the corresponding unfused index in `schedule`. This is a stylistic choice and we could have chosen a different name.
+The fused iteration space `schedule` has a shape of (2, 16, 10, 11). Its slice (0, \*, \*, \*) contains a copy of `schedule0`, the slice (1, \*, \*, 0) contains a copy of `schedule1`, and the rest of its elements are padded. Note that the code above overwrites the index `k0`, which initially was an index of `schedule0`. However, now it corresponds to the unfused index in `schedule`. Note that this is a stylistic choice, and we could have chosen a different name.
```

Comment:
"Note that the name `k0` is a stylistic choice, we could have chosen a different name."